### ISSUE TITLE
Does not get the ip address

### ISSUE THREAD

**Issue by ewgenw** [karawin/Ka-Radio32#281]
karadio32 on esp32-wroom board does not get ip address from dhcp . in telnet i get last message "E (4010) event: invalid static ip" and it's all. 
On my wifi access point i see connected board with address 0.0.0.0 . 
When i write static ip - board write in telnet that address is which I entered, but there is no connection , and acess point show address 0.0.0.0

**Comment by karawin** [comment]
Please read the documentation.
The first time KaRadio starts in AP mode to be able to configure the wifi.

**Comment by ewgenw** [comment]
Yes , i did it - in AP mode I have configured the name and password of my AP. After rebooting, the board connects to the AP, but does not receive an IP address.
this is how it looks on my AP
![IMG-1150abf3a53890c8f85cabcb0585d37d-V.jpg](https://user-images.githubusercontent.com/88132889/223806454-9f8acaf2-dcc2-46f7-8c2f-b75a9f90c85e.jpg)

And this is console of ESP board
![IMG-502ed6b12c7f22c76603455c3cb2ad9e-V.jpg](https://user-images.githubusercontent.com/88132889/223806797-4d20ebe5-2915-4e9a-b8ac-4a704ef11f6f.jpg)

**Comment by karawin** [comment]
DHCP checked on the interface?
